plugins/auth-backend/src/lib/resolvers/CatalogAuthResolverContext.ts

```diff
@@ -123,8 +123,17 @@ export class CatalogAuthResolverContext implements AuthResolverContext {
       const res = await this.catalogApi.getEntities({ filter }, { token });
       result = res.items;
     } else if ('filter' in query) {
+      const filter = [query.filter].flat().map(value => {
+        if (!('kind' in Object.keys(value).map(key => key.toLowerCase()))) {
```

Comment:
```suggestion
        if (!('kind' in Object.keys(value).map(key => key.toLocaleLowerCase('en-US')))) {
```

This actually leads to problems in e.g. Turkish locales otherwise.